charts/estafette-k8s-hpa-scaler/templates/rbac.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{.Chart.Name | quote}}
+  labels:
```

Comment:
I would add the `extraLabels` here and everywhere else as well, because it makes it easier to quickly find all resources for a certain label, regardless of resource type.

Reply:
agreed!